### PR TITLE
Properly fix legend issue in develop.

### LINKED ISSUE #1803: default wms server from config not accessible
besides the localhost builtin url the other two don't work
```
default_WMS = [
        "http://localhost:8081/",
        "http://eumetview.eumetsat.int/geoserver/wms",
        "https://apps.ecmwf.int/wms/?token=public"
    ]

```

### PR DESCRIPTION
**Pull request by joernu76**
Was fixed in stable. Other improper fixes in develop prevent this from working; also a merge error.

**Purpose of PR?**:

Fixes #1928

### CODE REVIEW

File: mslib/msui/wms_control.py [hunk 1]
```diff
@@ -1478,24 +1478,21 @@ def append_multiple_images(self, imgs):
         """
         images = [x for x in imgs if x]
         if images:
-            if hasattr(self.view, 'fig') is False:
```
Comment (ReimarBauer) [inline mslib/msui/wms_control.py:1481]: this was needed before not to crash on a different attribute than 'fig'

https://geoportal2.duisburg.de/arcgisserver/services/Luftbilder/Historische_Luftbilder/MapServer/WMSServer

now it crashes on 

```
Fatal error in MSS 8.3.0 on Linux-6.2.0-10018-tuxedo-x86_64-with-glibc2.35
Python 3.10.8 | packaged by conda-forge | (main, Nov 22 2022, 08:23:14) [GCC 10.4.0]

Please report bugs in MSS to https://github.com/Open-MSS/MSS

Information about the fatal error:

Traceback (most recent call last):
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1383, in continue_retrieve_image
    self.display_retrieved_image(img, legend_img, layer, style, init_time, valid_time, complete_level)
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1629, in display_retrieved_image
    self.view.draw_legend(self.append_multiple_images(legend_imgs))
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1484, in append_multiple_images
    max_height = int((self.view.plotter.fig.get_size_inches() * self.view.plotter.get_dpi())[1] * 0.99)
AttributeError: 'TopViewPlotter' object has no attribute 'get_dpi'
```


see also
https://github.com/Open-MSS/MSS/issues/1803

**Review by ReimarBauer — COMMENTED** [review]
see comment, I don't know why it now "crashes" differently

**Comment by joernu76** [comment]
Because the actual fix was not properly merged from stable and I botched the manual reapplication. Works now.

File: mslib/msui/mscolab.py [hunk 2]
```diff
@@ -1412,7 +1412,7 @@ def render_new_permission(self, op_id, u_id):
                 operation_desc = f'{operation["path"]} - {operation["access_level"]}'
                 widgetItem = QtWidgets.QListWidgetItem(operation_desc, parent=self.ui.listOperationsMSC)
                 widgetItem.op_id = operation["op_id"]
-                widgetItem.catgegory = operation["category"]
+                widgetItem.operation_category = operation["category"]
```
Comment (ReimarBauer) [inline mslib/msui/mscolab.py:1415]: here and also in stable I wonder what it means that we don't have seen a failure popping up

Comment (joernu76) [reply mslib/msui/mscolab.py:1415]: Apparently, you can assign whatever you want as an attribute (yeah, duck-typing!)

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**